test/e2e/alias_test.go

```diff
@@ -15,14 +15,26 @@ limitations under the License.
 
 package e2e
 
-import "knative.dev/eventing/test/common"
-import "knative.dev/eventing/test"
+import (
+	"os"
+	"testing"
+
+	"knative.dev/eventing/test"
+	"knative.dev/eventing/test/common"
+)
 
 var setup = common.Setup
 var tearDown = common.TearDown
 var getChannelTypeMeta = common.GetChannelTypeMeta
-var channels = test.EventingFlags.Channels
-var channelTestRunner = common.ChannelTestRunner{
-	ChannelFeatureMap: common.ChannelFeatureMap,
-	ChannelsToTest:    test.EventingFlags.Channels,
+var channels test.Channels
+var channelTestRunner common.ChannelTestRunner
+
+func TestMain(m *testing.M) {
```

Comment:
Really? There's no cleaner fix other than wrapping initialization in a TestMain?

Reply:
That's what Chao told me to do, since I will need to use the flag before running the actual tests.